pages/learn/manage/support-access.md

```diff
@@ -19,10 +19,10 @@ To enable support access for a single device, select the _Actions_ menu in the D
 
 ![Grant support for a device](/img/common/support/enable-support-access-device.gif)
 
-## Grant support access for an application
+## Grant support access for a fleet
 
-To enable support access for all devices in an application, select the _Grant Support Access_ from the _Actions_ menu of the Application dashboard, and choose the period to grant access. This may be revoked at any time by selecting _Revoke Support Access_ on the same page.
+To enable support access for all devices in a fleet, select the _Grant Support Access_ from the _Actions_ menu of the Fleet dashboard, and choose the period to grant access. This may be revoked at any time by selecting _Revoke Support Access_ on the same page.
```

Comment:
```suggestion
To enable support access for all devices in a fleet, select the _Grant Support Access_ from the _Settings_ menu of the Fleet dashboard, and choose the period to grant access. This may be revoked at any time by selecting _Revoke Support Access_ on the same page.
```